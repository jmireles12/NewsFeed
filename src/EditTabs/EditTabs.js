import React, { Component } from 'react'
import ApiContext from '../ApiContext'
import TabName from '../TabName/TabName'

export default class EditTabs extends Component {
    
    static defaultProps = {
        match: {
            params: {}
        }
    }

    static contextType = ApiContext;

    render() {
        const { tabs=[] } = this.context
        return (
            <section className='EditTabs'>
                <ul className='EditTabs__ul'>
                    {tabs.map(tab =>
                        <li className='EditTabs__li' key={tab.id} value={tab.id}>
                            <TabName 
                                id={tab.id}
                                name={tab.name}
                            />
                        </li>)}
                </ul>
                
            </section>
        )
    }
}