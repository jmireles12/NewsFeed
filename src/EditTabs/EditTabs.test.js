import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import EditTabs from './EditTabs'

describe(`EditTabs component`, () => {
    it('renders the complete EditTabs', () => {
        const wrapper = shallow(<EditTabs />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})