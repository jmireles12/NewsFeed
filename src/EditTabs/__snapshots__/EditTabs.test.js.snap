// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditTabs component renders the complete EditTabs 1`] = `
<section
  className="EditTabs"
>
  <ul
    className="EditTabs__ul"
  />
</section>
`;
